### PR TITLE
Add NativeScript Core XML Syntax

### PR DESCRIPTION
**Pull request by CatchABus**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add key bindings. **
- [x] If my package is a syntax it doesn't also add a color scheme. ***

My package is [NativeScript Core XML Syntax](https://github.com/CatchABus/sublime-nativescript-xml-syntax).
There are no packages like it in Package Control.

This package adds syntax highlighting for [NativeScript Core](https://docs.nativescript.org/tutorial/plain.html#getting-started-with-nativescript) XML files.

Syntax is identical to the default one for XML with the addition of highlighting view model bindings.
The highlight color is the one used for CDATA tags.

### CODE REVIEW

**Comment by CatchABus** [comment]
@packagecontrol-bot

**Comment by CatchABus** [comment]
@packagecontrol-bot

**Comment by CatchABus** [comment]
@packagecontrol-bot

**Comment by CatchABus** [comment]
@packagecontrol-bot

File: repository/n.json [hunk 1]
```diff
@@ -111,6 +111,17 @@
 				}
 			]
 		},
+		{
+			"name": "NativeScript Core XML Syntax",
```
Comment (braver) [inline repository/n.json:115]: Language support packages can just be named after the language, without suffixes like "syntax", "language", "highlighting" etc. In this case I would lean towards "NativeScript XML"?

**Comment by braver** [comment]
Have you considered extending or including the default XML syntax instead of duplicating most of it?

Your main scope is `text.xml`, but since you're extending the basic syntax for that, you might want to have a more specific one like `text.xml.nativescript`. Everything that supports  `text.xml` would also support your syntax, but yours can now be targeted specifically as well.

I'm a bit worried about this replacing the default XML syntax for many users, for files that are not necessarily NativeScript. May or may not be a problem, especially if this is truly an extension of the default and maybe only for specific file extensions.

File: repository/n.json (hunk 1 above)
Comment (CatchABus) [reply repository/n.json:115]: NativeScript is a mobile framework with multiple flavors (NativeScript Vue, Svelte, etc). NativeScript Core is a flavor itself, hence the `NativeScript Core` reference.

NativeScript Core consists of JS, CSS, and XML files. In XML, there is a custom expression syntax where attribute values are enclosed in double curly braces (e.g. `{{ myVariable }}`), that is why I focus on syntax.

**Comment by CatchABus** [comment]
> Have you considered extending or including the default XML syntax instead of duplicating most of it?
> 
> Your main scope is `text.xml`, but since you're extending the basic syntax for that, you might want to have a more specific one like `text.xml.nativescript`. Everything that supports `text.xml` would also support your syntax, but yours can now be targeted specifically as well.
> 
> I'm a bit worried about this replacing the default XML syntax for many users, for files that are not necessarily NativeScript. May or may not be a problem, especially if this is truly an extension of the default and maybe only for specific file extensions.

NativeScript is an existing framework that makes uses of XML files for UI rendering stuff.
This is currently a really small plugin that adds support for highlighting double-curly brace expressions `{{ }}` so it's a very small addition for xml in common. I can understand if it's a problem.

I could find a solution when plugin contains more features that affect the general use of XML.

File: repository/n.json (hunk 1 above)
Comment (braver) [reply repository/n.json:115]: Alright, makes sense then 👍🏻

**Pull request merged**